css/interactive-guides/circuit-breaker/circuit-breaker.scss

```diff
@@ -264,10 +264,11 @@
 .teContainer>.nav-tabs>li>a {
   width: 100% !important;
   line-height: 26px !important;
-  color: #3f465a !important;
-  background-color: #c7ccd9;
 }
 
 .teContainer>.teTabs>li {
-  width: 20% !important;
+  @media (min-width: 768px) {
+    width: 20% !important;
```

Comment:
What about the following code in tabbedEditor.js?  
![image](https://user-images.githubusercontent.com/29490139/46818548-8dce0600-cd4f-11e8-9e11-fce521185cf9.png)

Perhaps this is what needs to be adjusted and/or removed so that you don't have to set the css width as '!important'.